heft-plugins/heft-dev-cert-plugin/src/DevCertPlugin.ts

```diff
@@ -71,30 +71,55 @@ export class DevCertPlugin implements IHeftPlugin {
             await this._configureDevServerAsync(
               bundleSubstage.properties?.webpackConfiguration as IWebpackConfigPartial | undefined,
               certificateManager,
-              logger
+              logger,
+              this._determineMajorVersion(bundleSubstage.properties.webpackDevServerVersion)
             );
           });
         }
       });
     });
   }
 
+  private _determineMajorVersion(version?: string): number | undefined {
```

Comment:
I'm not crazy about these means to simplify major version detection, so I'm open for improvements/suggestions here.

Reply:
You could literally just do `parseInt(version)`, since `parseInt` allows for trailing non-digit values (and ignores them).

Reply:
Awesome, updated it to use parseInt instead.